client/rest-high-level/src/test/java/org/elasticsearch/client/SecurityRequestConvertersTests.java

```diff
@@ -101,6 +102,19 @@ public void testDeleteUser() {
         assertNull(request.getEntity());
     }
 
+    public void testGetUsers() {
+        final String[] users = new String[] {"test"};
+        GetUsersRequest getUsersRequest = new GetUsersRequest(users);
+        final RefreshPolicy refreshPolicy = randomFrom(RefreshPolicy.values());
+        final Map<String, String> expectedParams = getExpectedParamsFromRefreshPolicy(refreshPolicy);
+        Request request = SecurityRequestConverters.getUsers(getUsersRequest);
+        assertEquals(HttpGet.METHOD_NAME, request.getMethod());
+        assertEquals("/_xpack/security/user/test", request.getEndpoint());
```

Comment:
`/_xpack/security/user/test` - > `/_security/user/test` since https://github.com/elastic/elasticsearch/pull/36293 and https://github.com/elastic/elasticsearch/pull/36379 are merged

Reply:
:+1: